### DISCUSSION TITLE
How to style specific focused/highlighted  element ?

### DISCUSSION THREAD

**Discussion by sohitkumar** [elrumordelaluz/reactour#566] [General]
hi team, 

I have stucked in achieving one requirement, I want to style specific focused element like want to increase the width, height padding of specific highlighted element.
Is there any way to achieve this ?

**Comment by elrumordelaluz** [comment]
Hi @sohitdeveloper, the tour does not touch the DOM element you are highlighting, creates a _Mask_ focusing the element. You can use the `useTour` hook to know which is the `currentStep` to style the elemento on your side.
